examples/websocket-relay/relay-app/src/components/node/Node.svelte

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
+  import { Anchor, Node } from "svelvet";
+  import type { CSSColorString } from "svelvet";
+
+  import { base64CatStore, edgeStore, firstWorkflowToRunStore } from "../../stores";
+  import type { Position } from "$lib/node";
+  import type { Task } from "$lib/task";
+  import Edge from "./Edge.svelte";
+
+  export let base64Cat: string = $base64CatStore;
+  export let bgColor: CSSColorString = "white";
```

Comment:
Oh wut? Didn't know about the `CSSColorString` type. That's cool.

Reply:
Yeah! It comes from the `svelvet` lib 👌🏼